### PR TITLE
Allow parsing class constants with more than one wildcard in the constant name

### PR DESCRIPTION
**Pull request by Seldaek**
Refs phpstan/phpstan#5534

### CODE REVIEW

File: src/Parser/ConstExprParser.php [hunk 1]
```diff
@@ -61,10 +61,18 @@ public function parse(TokenIterator $tokens, bool $trimStrings = false): Ast\Con
 					$tokens->consumeTokenType(Lexer::TOKEN_IDENTIFIER);
 					if ($tokens->tryConsumeTokenType(Lexer::TOKEN_WILDCARD)) {
 						$classConstantName .= '*';
+						if ($tokens->currentTokenType() === Lexer::TOKEN_IDENTIFIER) {
```
Comment (ondrejmirtes) [inline src/Parser/ConstExprParser.php:64]: I feel like this needs a `while` loop - so that we can have multiple wildcards in the name :)

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: src/Parser/ConstExprParser.php (hunk 1 above)
Comment (Seldaek) [reply src/Parser/ConstExprParser.php:64]: OK sure, might as well do a complete solution while I'm at it I guess.

Comment (ondrejmirtes) [reply src/Parser/ConstExprParser.php:64]: Yes, thank you :)

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**